packages/components/codemods/Well/transformWell.spec.ts

```diff
@@ -0,0 +1,186 @@
+      import { Well } from "@kaizen/draft-well"
+      export const TestComponent = () => <div><Well>Test</Well></div>`)
+    const output = parseJsx(`
+      import { Well } from "@kaizen/draft-well"
+      export const TestComponent = () => <div><Well>Test</Well></div>`)
+    const transformed = transformWellSource(input)
+    expect(printAst(transformed).trim()).toBe(printAst(output).trim())
+  })
+
+  it('should replace variant="default" with color="gray"', () => {
+    const inputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well variant="default">Test</Well>
+      `)
+    const outputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well color="gray">Test</Well>
+    `)
+    const transformed = transformWellSource(inputAst)
+    expect(printAst(transformed).trim()).toBe(printAst(outputAst).trim())
+  })
+
+  it('should replace variant="informative" with color="blue"', () => {
+    const inputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well variant="informative">Test</Well>
+    `)
+    const outputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well color="blue">Test</Well>
+    `)
+    const transformed = transformWellSource(inputAst)
+    expect(printAst(transformed).trim()).toBe(printAst(outputAst).trim())
+  })
+
+  it('should add color="gray" if variant is not specified', () => {
+    const inputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well>Test</Well>
+    `)
+    const outputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well color="gray">Test</Well>
+    `)
+    const transformed = transformWellSource(inputAst)
+    expect(printAst(transformed).trim()).toBe(printAst(outputAst).trim())
+  })
+
+  it("should handle multiple attributes and replace only variant", () => {
+    const inputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well variant="informative" id="123">Test</Well>
+    `)
+    const outputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well color="blue" id="123">Test</Well>
+    `)
+    const transformed = transformWellSource(inputAst)
+    expect(printAst(transformed).trim()).toBe(printAst(outputAst).trim())
+  })
+
+  it("should handle multiple attributes and add color if variant is not specified", () => {
+    const inputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well id="123">Test</Well>
+    `)
+    const outputAst = parseJsx(`
+      import {Well} from "@kaizen/components"
+      export const TestComponent = () => <Well id="123"  color="gray">Test</Well>
```

Comment:
Does it fail if we remove the extra space?
```suggestion
      export const TestComponent = () => <Well id="123" color="gray">Test</Well>
```

Reply:
works either way but worth cleaning up the code. I think through the process of running through the transformer it might remove whitespace?